pom.xml

```diff
@@ -1098,12 +1068,26 @@
                   <artifactId>maven-antrun-plugin</artifactId>
                   <version>3.1.0</version>
                   <executions>
+                      <execution>
+                          <id>get-buildvers</id>
+                          <goals>
+                              <goal>run</goal>
+                          </goals>
+                          <configuration>
+                              <target xmlns:ac="antlib:net.sf.antcontrib">
+                                  <property name="expression" value="${expression}"/>
+                                  <script language="jython" src="${dyn.shim.rundir}/build/dyn_shim_detection.py"/>
```

Comment:
It's unclear to me why we need to execute the script twice, once here and once in the execution block below?

Reply:
It's only called once during our build. The script declared on line 1072 is only declared there to be called by the `version-def.sh`